guide/14-deep-learning/object_tracking.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
<p>It would be better if we can add a figure here to better illustrate the idea. Most people in GIS are new to this. Even adding the citation to some sort of scientific paper or video could be helpful</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/549/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>